index.bs

```diff
@@ -696,6 +698,39 @@ authorizing an authenticator with which to complete the operation.
 </div>
 
 
+### Signal platform authenticator capability - isPlatformAuthenticatorReady() method ### {#isPlatformAuthenticatorReady}
+
+<div link-for-hint="WebAuthentication/isPlatformAuthenticatorReady">
+This method is used to signal users whether they can log in to the [RP] with the <a>platform-attached</a> authenticator. 
```

Comment:
"Whether they can log in" implies that it tells the user a yes/no answer. In reality this is (potentially, if the platform is capable) telling the user that the RP would like to help them set up their platform authenticator.